test/get-workspaces-paths-test.js

```diff
@@ -166,7 +166,10 @@ describe(function() {
         fixturify.writeSync(tmpPath, {
           'packages': {
             'package-a': {
-              'package.json': stringifyJson({}),
+              'package.json': stringifyJson({
```

Comment:
Not sure why this was needed, but with the tests, we should duplicate all the tests using shouldSpawn: true, using the same setup code, and making sure the assertions are all the same. Then we can be sure we are matching the native implementation.

Reply:
makes sense